WordPress/Classes/ViewRelated/Reader/ReaderCrossPostCell.swift

```diff
@@ -3,16 +3,23 @@ import Foundation
 import AutomatticTracks
 import WordPressShared.WPStyleGuide
 
+private struct Constants {
+    static let imageBorderWidth: CGFloat = 1
```

Comment:
Do we want 1pt or "1px" border? Ref: https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Extensions/UIView+Borders.swift#L53

Reply:
I originally tried to use it, but from my experience using the hairline width it results in a 0.5px border instead of a 1px border which makes it nearly invisible.